INSTALL.rst

```diff
@@ -0,0 +1,68 @@
+..
+    Copyright (C) 2018 CERN.
+
+    Asclepias Broker is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Installation
+============
+
+To run an instance of ``Asclepias Broker`` you will need a PostgreSQL and Elasticsearch, which can be installed directly on your machine,
+or you can use ``docker-compose`` with the provided docker configuration to run those in the sandboxed docker environment (recommended).
+
+Quick start
+-----------
+
+.. note::
+
+    The docker configuration provided in this module uses PostgresSQL and ElasticSearch 6.
+
+Install ``docker`` and ``docker-compose`` in your machine.
+Then, create and run the services using the docker containers:
+
+.. code-block:: console
+
+    $ cd asclepias-broker
+    $ docker-compose up db es kibana
```

Comment:
This requires a certain minimum version of docker that should be specified. I originally tried doing it with Docker 13.x, but got:

```
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version ("2.0", "2.1", "3.0") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
```

Upgraded to 18 and it worked. So I think it would be good to specify the minimum version here somewhere.

Reply:
It would also be good to say very briefly what ``up``, ``db``, etc. are and what services they refer to.